modules/openapi-generator/src/main/resources/dart/libraries/dio/pubspec.mustache

```diff
@@ -4,7 +4,7 @@ description: {{pubDescription}}
 homepage: {{pubHomepage}}
 
 environment:
-  sdk: '>=2.15.0 <3.0.0'
+  sdk: '>=2.17.0 <3.0.0'
```

Comment:
Any specific reason for this bump?

Reply:
Ahn nvm, the enum instance variables. I don't mind this but we need to to this in at least a minor version.